packages/sync-actions/test/product-sync-variants.spec.js

```diff
@@ -291,42 +291,186 @@ describe('Actions', () => {
     ])
   })
 
-  it('should handle mapping actions for new variants without masterVariant',
-  () => {
-    const before = {
-      id: '123',
-      version: 1,
-      masterVariant: {
-        id: 1,
-        sku: 'v1',
-        attributes: [{ name: 'foo', value: 'bar' }],
-      },
-      variants: [
-        { id: 2, sku: 'v2', key: 'v2', attributes: [{ name: 'foo', value: 'qux' }] },
-        { id: 3, sku: 'v3', key: 'v3', attributes: [{ name: 'foo', value: 'baz' }] },
-      ],
-    }
+  describe('without master variant in `now`', () => {
+    describe('with master variant in `before`', () => {
```

Comment:
This tries to BDD down the tests

Reply:
Personally I think the test description is not explanatory enough

A sample below
```
Actions › without master variant in `now` › with master variant in `before` › should not generate update action to remove master variant
```

compared to something like this

```
Actions › should build `addVariant` action
```

No hard opinion on this though. Usable for me